pkg/server/v3/delta_test.go

```diff
@@ -457,3 +458,95 @@ func TestDeltaCallbackError(t *testing.T) {
 		})
 	}
 }
+
+// BENCHMARKS =====================================================================================================
+
+func BenchmarkDeltaResponseHandlers(b *testing.B) {
```

Comment:
What logic is this test testing? It's a bit hard from reading the code to understand what is mocked and what is not

Reply:
This is testing the server code for delta. When a stream is created, receives a request, responds, etc...

Reply:
I think I'm a bit unclear on how accurate this test would be since we're having to spin up a goroutine per test to process it. Maybe for now just document what this does and leave any improvements to later iterations?